dbfit-java/build.gradle

```diff
@@ -222,3 +251,114 @@ project('netezza') {
     }
 }
 
+// Aggregated dependencies to include in the POM file when publishing.
+dependencies {
+    compile publicCoreCompileDeps
+    runtime publicHsqldbRuntimeDeps
+    compile publicOracleCompileDeps
+    compile publicMysqlCompileDeps
+    compile publicPostgresCompileDeps
+}
+
+// Remove the default artifact.
```

Comment:
> perhaps I should read the docs at some point :-)):

Me too. There does seem to be endless ways in which to do any one thing in Gradle. Very flexible but having not really sat down to learn it properly yet it feels a bit *hit and miss* still.

> `configurations.archives.artifacts.clear()`

Looks good to me!